cloud/pkg/controllermanager/edgeapplication/overridemanager/imageparser/imageparser.go

```diff
@@ -153,14 +153,16 @@ func Parse(image string) (*Components, error) {
 	}
 
 	comp := &Components{}
-	if named, ok := ref.(reference.Named); ok {
+	switch named := ref.(type) {
+	case reference.Named:
 		comp.hostname, comp.repository = SplitHostname(named.Name())
 	}
```

Comment:
Only one case, seems have no difference with if..

Reply:
Yes, if it's just if, there's really no need to use the switch statement, I've changed it.